src/plone.server/plone/server/api/__init__.py

```diff
@@ -0,0 +1,12 @@
+# these imports are done to force loading services
+from . import addons  # noqa
+from . import app  # noqa
+from . import file  # noqa
```

Comment:
``file`` is a python keyword, it would be wise to rename it to something else at some point, just to prevent hard to debug errors and the likes